tests/e2e/pact/specs/live/validate.spec.ts

```diff
@@ -0,0 +1,69 @@
+import * as jestpact from "jest-pact"
+import * as uuid from "uuid"
+import {basePath, pactOptions} from "../../resources/common"
+import supertest from "supertest"
+import {fetcher} from "@models"
+import {InteractionObject} from "@pact-foundation/pact"
+import * as LosslessJson from "lossless-json"
+
+jestpact.pactWith(
+  pactOptions("live", "validate"),
+  /* eslint-disable  @typescript-eslint/no-explicit-any */
+  async (provider: any) => {
+    const client = () => {
+      const url = `${provider.mockService.baseUrl}`
+      return supertest(url)
+    }
+
+    describe("validate e2e tests", () => {
+      test("validate endpoint should return 200 on success", async () => {
+        const testCase = fetcher.convertExamples[0]
+        const apiPath = `${basePath}/$validate`
+        const requestId = uuid.v4()
+        const correlationId = uuid.v4()
+
+        const messageStr = LosslessJson.stringify(testCase.request)
+
+        const interaction: InteractionObject = {
+          state: "is authenticated",
+          uponReceiving: "a valid FHIR message",
+          withRequest: {
+            headers: {
+              "Content-Type": "application/fhir+json; fhirVersion=4.0",
+              "X-Request-ID": requestId,
+              "X-Correlation-ID": correlationId
+            },
+            method: "POST",
+            path: apiPath,
+            body: LosslessJson.parse(messageStr)
```

Comment:
Will need to be JSON parse here, as lossless parse adds extra fields resulting in mismatch between expected and actual

Reply:
just saw the failure for this lol, fixing